component/Image/image.css

```diff
@@ -0,0 +1,15 @@
+.responsive-image img {
+	width: 100%;
+	height: auto;
```

Comment:
Instead of assigning these class name to div, assign it to the image tag directly.

Reply:
I have put img tag in div because it doesn't follow the section container otherwise and flows into the page. Applying the classes to img itself makes the image not follow the section width.

Reply:
Okay